cli/load_restaurants

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+#
+#  load_restaurants
+#  Copyright(c) 2016 Bitergia
+#  Author: Bitergia <fiware-testing@bitergia.com>
+#  MIT Licensed
+#
+#  load restaurants subcommand for TourGuide CLI.
+#
+
+load_restaurants_wait=0
+restaurant_name=""
+limit=0
+feeder_params=""
+
+function submodule_help () {
+    cat <<EOF >&2
+Usage: ${appname} load restaurants [-h | --help] [-w | --wait] [-r <name> | --restaurant <name>]
+                                   [-l <n> | --limit <n>]
+
+Load sample restaurants for use with TourGuide-App.
+
+Command options:
+
+  -h  --help                 Show this help.
+  -w  --wait                 Wait for the tourguide container to be ready.
+                             Default is to exit if tourguide container is not ready.
+  -r  --restaurant <name>    Load a single restaurant using its <name>.
+  -l  --limit <n>            Limit the number of restaurants to load to <n>.
+                             Using --restaurant limits <n> to 1.
```

Comment:
We think that both suggestions are clearer. We prefer the second one (“This option is ignored when using --restaurant option.”). Thank you!